dev-data/react-ru.data.ts

```diff
@@ -1,8 +1,8 @@
+import type { Trainer } from '@/entities/trainer/types';
```

Comment:
violation of the `public api` rule
https://feature-sliced.design/docs/reference/public-api


<img width="888" alt="Screenshot 2024-09-07 at 22 09 21" src="https://github.com/user-attachments/assets/8e84e714-7820-4641-aa54-b86a1c55f2d5">


```suggestion
import type { Trainer } from '@/entities/trainer';
```

Reply:
fixed in https://github.com/rolling-scopes/site/pull/492/commits/17c6d49e2410fa5452cf36f8633121bed5cb87ba